unreal_asset/src/types/fname.rs

```diff
@@ -50,6 +50,29 @@ pub enum FName {
     },
 }
 
+impl std::fmt::Debug for FName {
+    fn fmt(&self, f: &mut std::fmt::Formatter<'_>) -> Result<(), std::fmt::Error> {
+        match self {
+            FName::Backed {
+                index,
+                number,
+                ty,
+                name_map,
+            } => f
+                .debug_struct("Backed")
+                .field("value", &name_map.get_ref().get_name_reference(*index))
+                .field("number", number)
+                .field("ty", ty)
+                .finish(),
```

Comment:
Maybe include some info about the name map? Not everything, maybe just it's size?